#40: Fixed source scope (crawler tried to crawl resources) (#41)
exasol/error-code-crawler-maven-plugin: doc/changes/changes_0.4.0.md

* #35: Removed hardcoded version numbers from tests
* #37: Renamed `errorCodeConfig.yml` to `error_code_config.yml`

## Bugfixes

> * #40: Fixed source scope (crawler tried to crawl resources)

Reviewer: ```suggestion
* #40: Fixed source scope so that it does not crawl resources
```
